src/EFCore.PG/Utilities/SortOrderHelper.cs

```diff
@@ -0,0 +1,48 @@
+using System.Linq;
+using JetBrains.Annotations;
+using Npgsql.EntityFrameworkCore.PostgreSQL.Metadata;
+
+namespace Npgsql.EntityFrameworkCore.PostgreSQL.Utilities
+{
+    static class SortOrderHelper
+    {
+        public static bool IsDefaultSortOrder([NotNull] SortOrder[] sortOrders)
+        {
+            Check.NotNull(sortOrders, nameof(sortOrders));
+
+            return sortOrders.All(sortOrder => sortOrder != SortOrder.Descending);
```

Comment:
nit: seems more natural to check `sortOrder == SortOrder.Ascending`

Reply:
Yeah, now that `Unspecified` is gone, that makes more sense. We'll spotted!